### ISSUE TITLE
roadmap: unjs articles

### ISSUE THREAD

**Issue by Barbapapazes** [unjs/website#110]
Hello 👋,

I open this PR to explain what's coming on the UnJS website. Since more than 4 weeks, we are rebuilding it from sractch to bring you something new. In fact, we want to make the UnJS website **more than just a showcase** for each package. We want to make it a real HQ for everyone.

- A blog, for users who want the latest update (related to package release)
- 3 resources sections
	- Learn, with 101 articles for users who want to learn the basic and get into the ecosystem
	- Build, to create more complex projects to show you how to play and plug packages together
	- Explore, to deep dive into some packages or functionalities

Resources will be articles (many), and this PR is to give you a roadmap for what's coming (could change anytime):

```[tasklist]
### Tasks
- [ ] ofetch 101
```

**Comment by pi0** [comment]
Would be nice to include unbuild as priority it has stable api and guide can benefit many authors imo!

**Comment by pi0** [comment]
Also for h3 since it lacks proper docs but even when we have them, a starter guide would be so useful!

**Comment by Barbapapazes** [comment]
> Would be nice to include unbuild as priority it has stable api and guide can benefit many authors imo!

Let's focus on this package!

**Comment by Barbapapazes** [comment]
> Also for h3 since it lacks proper docs but even when we have them, a starter guide would be so useful!

Yes, h3 is a big subject that will require more times (and maybe many articles). I'll take it after `unbuild`.
